flutter_src/lib/logic/state/team_state/team_notifier.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:collection';
+import 'package:flutter/material.dart';
+import 'package:macrobaseapp/model/entities/team.dart';
+
+class TeamNotifier with ChangeNotifier {
```

Comment:
Also similar to `macro_notifier.dart`

Reply:
Again generic class could be a solution here, but it is not worth here. Sometimes clarity is more important than concise code.